### PR TITLE
Update supported-data-types-in-carbondata.md

### PR DESCRIPTION
**Pull request by Chizhiwen160**
### Why is this PR needed?
 
 
 ### What changes were proposed in this PR?

    
 ### Does this PR introduce any user interface change?
 - No
 - Yes. (please explain the change and update document)

 ### Is any new testcase added?
 - No
 - Yes

### CODE REVIEW

File: docs/supported-data-types-in-carbondata.md [hunk 1]
```diff
@@ -1,29 +1,36 @@
 <!--
     Licensed to the Apache Software Foundation (ASF) under one or more 
     contributor license agreements.  See the NOTICE file distributed with
-    this work for additional information regarding copyright ownership. 
-    The ASF licenses this file to you under the Apache License, Version 2.0
+    this work for additional information reg you under the Apache License, Version 2.0
```
Comment (xubo245) [inline docs/supported-data-types-in-carbondata.md:4]: why do you change the license?

File: docs/supported-data-types-in-carbondata.md [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+
+
 #### CarbonData supports the following data types:
 
   * Numeric Types
     * SMALLINT
     * INT/INTEGER
     * BIGINT
-    * DOUBLE
+    * DOUBLETREE Hduusywnndbb.
```
Comment (xubo245) [inline docs/supported-data-types-in-carbondata.md:27]: what is this? Hduusywnndbb.

File: docs/supported-data-types-in-carbondata.md [hunk 3]
```diff
@@ -52,3 +58,32 @@
...
+z
+x
+zhdywikkb&^#<#
+
+djhdd77<@_//#&*#
+suduysyqw<<77#<&#(sduhsuduhn  
+
+
+  
+didhdj8872/___×_6677_hhxfjufhffddufxkx
```
Comment (xubo245) [inline docs/supported-data-types-in-carbondata.md:73]: why do you add this?

**Comment by chenliang613** [comment]
@Chizhiwen160  can you please fix this pr as per xubo comments.

**Comment by chenliang613** [comment]
@Chizhiwen160  because you didn't response, will close this PR

File: docs/supported-data-types-in-carbondata.md (hunk 2 above)
Comment (Chizhiwen160) [reply docs/supported-data-types-in-carbondata.md:27]: jjxhsnn 'mjjzhzjjn'xnnzjjzjjnhzgdhhsnnznzk&&.&/&/72¥.'zi8kkKkzmnmn¥/&/&&/'nzjjhjxhshsbs

File: docs/supported-data-types-in-carbondata.md [hunk 4]
```diff
@@ -52,3 +58,32 @@
...
+
+ssuxidjuhhdhh77x7j////-_
+
+s
+
+suus7s
+
```
Comment (Chizhiwen160) [inline docs/supported-data-types-in-carbondata.md:80]: a@iz

skas
zjkzmsiiusuzj7/¥&/'m'hnz
ssjjsjjzn
¥&!.'z
hhszsuislf
f
jdjjkdddjjd

xxjjxjjxhxhjxeidujdjxysmxmxm
xjxjjxhuusjwjwmxuxjjs

**Pull request closed without merging**